// @mdxeditor/editor/dist/styles/ui.module.css.js
const editorRoot = "_editorRoot_f3hmk_53";
const editorWrapper = "_editorWrapper_f3hmk_155";
const toolbarRoot = "_toolbarRoot_f3hmk_162";
const readOnlyToolbarRoot = "_readOnlyToolbarRoot_f3hmk_189";
const toolbarToggleItem = "_toolbarToggleItem_f3hmk_208";
const toolbarGroupOfGroups = "_toolbarGroupOfGroups_f3hmk_219";
const toolbarToggleSingleGroup = "_toolbarToggleSingleGroup_f3hmk_224";
const toolbarButton = "_toolbarButton_f3hmk_239";
const activeToolbarButton = "_activeToolbarButton_f3hmk_275";
const toolbarButtonDropdownContainer = "_toolbarButtonDropdownContainer_f3hmk_286";
const selectContainer = "_selectContainer_f3hmk_288";
const selectItem = "_selectItem_f3hmk_301";
const toolbarButtonSelectTrigger = "_toolbarButtonSelectTrigger_f3hmk_307";
const selectTrigger = "_selectTrigger_f3hmk_308";
const selectDropdownArrow = "_selectDropdownArrow_f3hmk_373";
const contentEditable = "_contentEditable_f3hmk_379";
const codeMirrorWrapper = "_codeMirrorWrapper_f3hmk_391";
const codeMirrorToolbar = "_codeMirrorToolbar_f3hmk_408";
const propertyPanelTitle = "_propertyPanelTitle_f3hmk_430";
const propertyEditorTable = "_propertyEditorTable_f3hmk_438";
const iconButton = "_iconButton_f3hmk_456";
const readOnlyColumnCell = "_readOnlyColumnCell_f3hmk_461";
const buttonsFooter = "_buttonsFooter_f3hmk_474";
const propertyEditorInput = "_propertyEditorInput_f3hmk_480";
const primaryButton = "_primaryButton_f3hmk_506";
const secondaryButton = "_secondaryButton_f3hmk_507";
const smallButton = "_smallButton_f3hmk_522";
const dialogForm = "_dialogForm_f3hmk_534";
const linkDialogEditForm = "_linkDialogEditForm_f3hmk_540";
const linkDialogInputContainer = "_linkDialogInputContainer_f3hmk_548";
const linkDialogInputWrapper = "_linkDialogInputWrapper_f3hmk_554";
const linkDialogInput = "_linkDialogInput_f3hmk_548";
const linkDialogAnchor = "_linkDialogAnchor_f3hmk_586";
const linkDialogPopoverContent = "_linkDialogPopoverContent_f3hmk_600";
const tableColumnEditorPopoverContent = "_tableColumnEditorPopoverContent_f3hmk_601";
const dialogContent = "_dialogContent_f3hmk_602";
const largeDialogContent = "_largeDialogContent_f3hmk_614";
const dialogTitle = "_dialogTitle_f3hmk_624";
const dialogCloseButton = "_dialogCloseButton_f3hmk_630";
const popoverContent = "_popoverContent_f3hmk_637";
const popoverArrow = "_popoverArrow_f3hmk_649";
const linkDialogPreviewAnchor = "_linkDialogPreviewAnchor_f3hmk_653";
const tooltipTrigger = "_tooltipTrigger_f3hmk_676";
const tooltipContent = "_tooltipContent_f3hmk_680";
const actionButton = "_actionButton_f3hmk_694";
const primaryActionButton = "_primaryActionButton_f3hmk_701";
const tableEditor = "_tableEditor_f3hmk_713";
const toolCell = "_toolCell_f3hmk_724";
const tableColumnEditorTrigger = "_tableColumnEditorTrigger_f3hmk_743";
const addRowButton = "_addRowButton_f3hmk_745";
const addColumnButton = "_addColumnButton_f3hmk_746";
const tableColumnEditorToolbar = "_tableColumnEditorToolbar_f3hmk_788";
const toggleGroupRoot = "_toggleGroupRoot_f3hmk_801";
const tableToolsColumn = "_tableToolsColumn_f3hmk_819";
const leftAlignedCell = "_leftAlignedCell_f3hmk_828";
const rightAlignedCell = "_rightAlignedCell_f3hmk_832";
const centeredCell = "_centeredCell_f3hmk_836";
const dialogOverlay = "_dialogOverlay_f3hmk_869";
const focusedImage = "_focusedImage_f3hmk_915";
const imageWrapper = "_imageWrapper_f3hmk_919";
const editImageToolbar = "_editImageToolbar_f3hmk_932";
const editImageButton = "_editImageButton_f3hmk_936";
const inlineEditor = "_inlineEditor_f3hmk_942";
const blockEditor = "_blockEditor_f3hmk_951";
const nestedEditor = "_nestedEditor_f3hmk_960";
const genericComponentName = "_genericComponentName_f3hmk_979";
const diffSourceToggleWrapper = "_diffSourceToggleWrapper_f3hmk_985";
const ggDiffSourceToggle = "_ggDiffSourceToggle_f3hmk_993";
const diffSourceToggle = "_diffSourceToggle_f3hmk_985";
const selectWithLabel = "_selectWithLabel_f3hmk_1011";
const toolbarTitleMode = "_toolbarTitleMode_f3hmk_1026";
const imageControlWrapperResizing = "_imageControlWrapperResizing_f3hmk_1031";
const imageResizer = "_imageResizer_f3hmk_1035";
const imageResizerN = "_imageResizerN_f3hmk_1044";
const imageResizerNe = "_imageResizerNe_f3hmk_1050";
const imageResizerE = "_imageResizerE_f3hmk_1056";
const imageResizerSe = "_imageResizerSe_f3hmk_1062";
const imageResizerS = "_imageResizerS_f3hmk_1062";
const imageResizerSw = "_imageResizerSw_f3hmk_1074";
const imageResizerW = "_imageResizerW_f3hmk_1080";
const imageResizerNw = "_imageResizerNw_f3hmk_1086";
const imagePlaceholder = "_imagePlaceholder_f3hmk_1092";
const imageDimensionsContainer = "_imageDimensionsContainer_f3hmk_1100";
const placeholder = "_placeholder_f3hmk_1105";
const rootContentEditableWrapper = "_rootContentEditableWrapper_f3hmk_1118";
const downshiftInputWrapper = "_downshiftInputWrapper_f3hmk_1128";
const downshiftInput = "_downshiftInput_f3hmk_1128";
const downshiftAutocompleteContainer = "_downshiftAutocompleteContainer_f3hmk_1159";
const textInput = "_textInput_f3hmk_1205";
const multiFieldForm = "_multiFieldForm_f3hmk_1213";
const formField = "_formField_f3hmk_1219";
const markdownParseError = "_markdownParseError_f3hmk_1230";
const popupContainer = "_popupContainer_f3hmk_1239";
const inputSizer = "_inputSizer_f3hmk_1244";
const styles = {
  editorRoot,
  editorWrapper,
  toolbarRoot,
  readOnlyToolbarRoot,
  toolbarToggleItem,
  toolbarGroupOfGroups,
  toolbarToggleSingleGroup,
  toolbarButton,
  activeToolbarButton,
  toolbarButtonDropdownContainer,
  selectContainer,
  selectItem,
  toolbarButtonSelectTrigger,
  selectTrigger,
  selectDropdownArrow,
  contentEditable,
  codeMirrorWrapper,
  codeMirrorToolbar,
  propertyPanelTitle,
  propertyEditorTable,
  iconButton,
  readOnlyColumnCell,
  buttonsFooter,
  propertyEditorInput,
  primaryButton,
  secondaryButton,
  smallButton,
  dialogForm,
  linkDialogEditForm,
  linkDialogInputContainer,
  linkDialogInputWrapper,
  linkDialogInput,
  linkDialogAnchor,
  linkDialogPopoverContent,
  tableColumnEditorPopoverContent,
  dialogContent,
  largeDialogContent,
  dialogTitle,
  dialogCloseButton,
  popoverContent,
  popoverArrow,
  linkDialogPreviewAnchor,
  tooltipTrigger,
  tooltipContent,
  actionButton,
  primaryActionButton,
  tableEditor,
  toolCell,
  tableColumnEditorTrigger,
  addRowButton,
  addColumnButton,
  tableColumnEditorToolbar,
  toggleGroupRoot,
  tableToolsColumn,
  leftAlignedCell,
  rightAlignedCell,
  centeredCell,
  dialogOverlay,
  focusedImage,
  imageWrapper,
  editImageToolbar,
  editImageButton,
  inlineEditor,
  blockEditor,
  nestedEditor,
  genericComponentName,
  diffSourceToggleWrapper,
  ggDiffSourceToggle,
  diffSourceToggle,
  selectWithLabel,
  toolbarTitleMode,
  imageControlWrapperResizing,
  imageResizer,
  imageResizerN,
  imageResizerNe,
  imageResizerE,
  imageResizerSe,
  imageResizerS,
  imageResizerSw,
  imageResizerW,
  imageResizerNw,
  imagePlaceholder,
  imageDimensionsContainer,
  placeholder,
  rootContentEditableWrapper,
  downshiftInputWrapper,
  downshiftInput,
  downshiftAutocompleteContainer,
  textInput,
  multiFieldForm,
  formField,
  markdownParseError,
  popupContainer,
  inputSizer
};
export {
  actionButton,
  activeToolbarButton,
  addColumnButton,
  addRowButton,
  blockEditor,
  buttonsFooter,
  centeredCell,
  codeMirrorToolbar,
  codeMirrorWrapper,
  contentEditable,
  styles as default,
  dialogCloseButton,
  dialogContent,
  dialogForm,
  dialogOverlay,
  dialogTitle,
  diffSourceToggle,
  diffSourceToggleWrapper,
  downshiftAutocompleteContainer,
  downshiftInput,
  downshiftInputWrapper,
  editImageButton,
  editImageToolbar,
  editorRoot,
  editorWrapper,
  focusedImage,
  formField,
  genericComponentName,
  ggDiffSourceToggle,
  iconButton,
  imageControlWrapperResizing,
  imageDimensionsContainer,
  imagePlaceholder,
  imageResizer,
  imageResizerE,
  imageResizerN,
  imageResizerNe,
  imageResizerNw,
  imageResizerS,
  imageResizerSe,
  imageResizerSw,
  imageResizerW,
  imageWrapper,
  inlineEditor,
  inputSizer,
  largeDialogContent,
  leftAlignedCell,
  linkDialogAnchor,
  linkDialogEditForm,
  linkDialogInput,
  linkDialogInputContainer,
  linkDialogInputWrapper,
  linkDialogPopoverContent,
  linkDialogPreviewAnchor,
  markdownParseError,
  multiFieldForm,
  nestedEditor,
  placeholder,
  popoverArrow,
  popoverContent,
  popupContainer,
  primaryActionButton,
  primaryButton,
  propertyEditorInput,
  propertyEditorTable,
  propertyPanelTitle,
  readOnlyColumnCell,
  readOnlyToolbarRoot,
  rightAlignedCell,
  rootContentEditableWrapper,
  secondaryButton,
  selectContainer,
  selectDropdownArrow,
  selectItem,
  selectTrigger,
  selectWithLabel,
  smallButton,
  tableColumnEditorPopoverContent,
  tableColumnEditorToolbar,
  tableColumnEditorTrigger,
  tableEditor,
  tableToolsColumn,
  textInput,
  toggleGroupRoot,
  toolCell,
  toolbarButton,
  toolbarButtonDropdownContainer,
  toolbarButtonSelectTrigger,
  toolbarGroupOfGroups,
  toolbarRoot,
  toolbarTitleMode,
  toolbarToggleItem,
  toolbarToggleSingleGroup,
  tooltipContent,
  tooltipTrigger
};
